### DISCUSSION TITLE
Display desktop icons right side of the screen

### DISCUSSION THREAD

**Discussion by m4vnel** [linuxmint/discussions#683] [Ideas]
It's possible to resize the desktop grid where icons live, but no put them on the right side of the screen. If this possibility doesn't involve a lot of work, I vote for it :-) It seems my left and right brain perception works differently, and perhaps users with right-to-left scripts would appreciate this feature. Thank you.

**Comment by 100savage** [comment]
If you right-click on the desktop and select "Customize," you can toggle the "Auto-arrange" switch to "Off." This will allow you to put the desktop icons wherever you like, including the right side of the screen.

**Comment by m4vnel** [comment]
Yes, I know, thank you. I meant a right side "Auto-arrange". If it's done manually, every new file I put in the desktop has to be placed "by hand", not are placed automatically in a grid, like occurs in the left.
